docs/en/stack/ml/df-analytics/ml-dfa-limitations.asciidoc

```diff
@@ -59,6 +59,21 @@ with the increased number of data points and how the quality of results scales
 with an increased training percentage.
 
 
+[float]
+[[dfa-restart]]
+== {dfanalytics-jobs-cap} may restart after an {es} upgrade
+  
+A {dfanalytics-job} may be restarted from the beginning in the following cases:
+
+* the job is in progress during an {es} update,
+* the job resumes on a node with a higher version,
+* the results field mapping has changed in the destination index.
```

Comment:
It might be clearer to reword this to "the results format has changed requiring different mappings in the destination index"

Reply:
@dimitris-athanasiou It is definitely clearer this way. I amended the text accordingly via d295874.